Editor/Scripts/MeshSyncServerInspector.cs

```diff
@@ -85,6 +86,34 @@ public void DrawServerSettings(MeshSyncServer t)
         }
     }
 
+    static void DrawDCCToolInfo(MeshSyncServer server)
+    {
+        if (server != null)
+        {
+            GUILayout.BeginHorizontal();
+
+            EditorGUI.BeginChangeCheck();
+
+            server.m_DCCAsset = EditorGUILayout.ObjectField("DCC asset file:", server.m_DCCAsset, typeof(UnityEngine.Object), true);
```

Comment:
The idea that a server can be associated with an asset kind of confuses me.
Can you elaborate why this is necessary ?

Reply:
It's optional to allow users to edit assets that are in Unity.

Reply:
So, referring to my previous comment [here](https://github.com/unity3d-jp/MeshSync/pull/697#issuecomment-1167002692)

If the purpose of this PR is to allow users to launch blender and let it open specific blend files under Assets folder, then they can just double click on blend files directly.
Why is it necessary to add a button in `MeshSyncServer` for this purpose ?

Reply:
It's just an easier way to live-edit.